README.md

```diff
@@ -100,139 +96,136 @@ Brave has published and implemented an [Ads Confirmation Protocol](https://githu
 Overview
 ========
 
-Impression Declaration
+Attribution Source Declaration
 ----------------------
 
-### Registering impressions for anchor tag navigations
+### Registering attribution sources for anchor tag navigations
 
-An impression tag is an anchor tag with special attributes:
+An attribution source is an anchor tag with special attributes:
 
-`<a conversiondestination="[eTLD+1]" impressiondata="[string]"
-impressionexpiry=[unsigned long long] reportingorigin="[origin]">`
+`<a attributeon="[eTLD+1]" attributionsourcedata="[string]"
```

Comment:
[PCM issue](https://github.com/privacycg/private-click-measurement/issues/30#issuecomment-730598490): `ID`
[CM issue](https://github.com/WICG/conversion-measurement-api/issues/57#issuecomment-743403963): `data`

Which one has been chosen?

Reply:
Good question. I think originally we were planning on having separate attribute names for the click side data in PCM and CM API. PCM went with id so this changed to data.

However, per the decision on https://github.com/privacycg/private-click-measurement/issues/59, it seems like we are going to have different endpoints. Because the endpoints are separate, we could choose to use the same attribute name, attributionsourceid.

Note that attributionsourcedata matches up somewhat more nicely with the aggregate API equivalents, as they do not store IDs.

+csharrison do you have any thoughts here?

Reply:
We should not use the same ID as PCM because otherwise it is difficult to interoperate (forces branching between the two browsers). Ideally we could get PCM to use "data" since it implies less entropy to me. Otherwise I liked "attribution-event-id" which was discussed in https://github.com/privacycg/private-click-measurement/issues/30